src/main/kotlin/com/terraformation/backend/accelerator/api/DeliverablesController.kt

```diff
@@ -0,0 +1,206 @@
+package com.terraformation.backend.accelerator.api
+
+import com.terraformation.backend.api.AcceleratorEndpoint
+import com.terraformation.backend.api.ApiResponse200
+import com.terraformation.backend.api.ApiResponse404
+import com.terraformation.backend.api.SuccessResponsePayload
+import com.terraformation.backend.db.accelerator.DeliverableCategory
+import com.terraformation.backend.db.accelerator.DeliverableId
+import com.terraformation.backend.db.accelerator.DeliverableType
+import com.terraformation.backend.db.accelerator.DocumentStore
+import com.terraformation.backend.db.accelerator.ParticipantId
+import com.terraformation.backend.db.accelerator.SubmissionDocumentId
+import com.terraformation.backend.db.accelerator.SubmissionStatus
+import com.terraformation.backend.db.default_schema.OrganizationId
+import com.terraformation.backend.db.default_schema.ProjectId
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.Parameter
+import io.swagger.v3.oas.annotations.headers.Header
+import io.swagger.v3.oas.annotations.media.Schema
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import java.net.URI
+import java.time.Instant
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@AcceleratorEndpoint
+@RequestMapping("/api/v1/accelerator/deliverables")
+@RestController
+class DeliverablesController {
+  @ApiResponse200
+  @GetMapping
+  @Operation(
+      summary = "Lists the deliverables for accelerator projects",
+      description =
+          "The list may optionally be filtered based on certain criteria as specified in the " +
+              "query string. If no filter parameters are supplied, lists all the deliverables " +
+              "in all the participants and projects that are visible to the user. For users with " +
+              "accelerator admin privileges, this will be the full list of all deliverables for " +
+              "all accelerator projects.")
+  fun listDeliverables(
```

Comment:
We will also need optional `category` and `status` query params. There is also a search bar in the FE table... originally built assuming usage of the Search API and doing a fuzzy search on `name` and `project_name` (I'll double check on `project_name`, both were added based on an assumption but the designs don't mention what the search bar should filter on). Do you think that should be added here or should we filter the results on the FE?

Reply:
My assumption was that for MVP the client would be grabbing the whole list with just the filters that were implied by the top nav and then doing the other filtering locally. I think that's how we do it in some other places in the UI. But it's easy to add those filter criteria here. Should I also add a way to change the sort order, or will we be doing client-side sorting?

Reply:
Does it need to support multiple values for category and/or status?

Reply:
That works for me, we can do all the filtering client side for now. I think we have generally always tried to leverage the API to do the filtering but in this case, for MVP, I think it is fine to do it client side. The sorting we can absolutely do client side, even after MVP. I think we should only consider server side sorting if we ever decide to implement pagination server side as well, otherwise we would be making requests that have no need for the server to compute the result of.

Reply:
> Does it need to support multiple values for category and/or status?

Yea the frontend uses multi select filters. But I think you've convinced me to just do it all in the frontend for now so you can disregard.

Reply:
We have been relying on server side sorting for i18n purposes. Especially if there are numbers involved, the display vs sort semantics. Guessing the number of documents won't be in the 1000s and we should be ok.
We do have client side code that merges two sets of data and does client side sorting, this works fine when we have both 'number' and 'number(raw)' in the payload.